src/test/java/org/junitpioneer/jupiter/AbstractEntryBasedExtensionTests.java

```diff
@@ -39,6 +40,13 @@ void setUp() {
 		System.setProperty(SET_SYSPROP_KEY, SET_SYSPROP_ORIGINAL_VALUE);
 	}
 
+	@AfterEach
+	void tearDown() {
+		// Revert changes to not affect other tests
```

Comment:
I think you can remove that comment.

Also, isn't it now possible to remove the clear calls from the `@BeforeEach`?

Reply:
The clear calls are for different variables though (`CLEAR_...`) whereas these this is for the `SET_...` variables. Or do you mean something else?